matchzoo/models/arcii_model.py

```diff
@@ -0,0 +1,124 @@
+"""An implementation of ArcII Model."""
+from matchzoo import engine
+from matchzoo import layers
+import typing
+import numpy as np
+from keras.models import Model
+from keras.layers import Input, Embedding, Conv1D, Conv2D, MaxPooling2D, \
+    Dropout, Flatten
+
+
+class ArcIIModel(engine.BaseModel):
+    """
+    ArcII Model.
+
+    Examples:
+        >>> model = ArcIIModel()
+        >>> model.guess_and_fill_missing_params()
+        >>> model.build()
+
+    """
+
+    @classmethod
+    def get_default_params(cls) -> engine.ParamTable:
+        """:return: model default parameters."""
+        params = super().get_default_params()
+        params['optimizer'] = 'adam'
+        params['input_shapes'] = [(32,), (32,)]
```

Comment:
`input_shapes` better to be left empty. It should be filled by either `guess_and_fill_missing_parameters` or manually by users.